.github/workflows/cli-tests.yaml

```diff
@@ -129,4 +129,10 @@ jobs:
         run: |
           export NIX_INSTALLER_NO_CHANNEL_ADD=1
           export DEVBOX_DEBUG=1
+          
+          # Setup github authentication to ensure Github's rate limits are not hit.
```

Comment:
What is hitting the limit? Nix install script?

Reply:
Print-dev-env on the flake.nix trying to fetch the GitHub:numtide package